zookeeper-client/zookeeper-client-c/src/zookeeper.c

```diff
@@ -1037,13 +1071,13 @@ int update_addrs(zhandle_t *zh)
         zh->state = ZOO_NOTCONNECTED_STATE;
     }
 
-fail:
+done:
```

Comment:
naming nit - "done" is technically accurate but doesn't capture the sense of an early finish. Alternatives?